tests/item/item_1d.cpp

```diff
@@ -24,32 +24,42 @@ class kernel_item_1d {
 
   t_readAccess m_x;
   t_writeAccess m_o;
+  cl::sycl::range<1> r_exp;
+  cl::sycl::id<1> offset_exp;
 
  public:
-  kernel_item_1d(t_readAccess in_, t_writeAccess out_) : m_x(in_), m_o(out_) {}
+  kernel_item_1d(t_readAccess in_, t_writeAccess out_, cl::sycl::range<1> r)
+      : m_x(in_), m_o(out_), r_exp(r), offset_exp(cl::sycl::id<1>(0)) {}
 
   void operator()(cl::sycl::item<1> item) const {
+    bool result = true;
+
     cl::sycl::id<1> gid = item.get_id();
 
     size_t dim_a = item.get_id(0);
     size_t dim_b = item[0];
+    result &= gid.get(0) == dim_a && gid.get(0) == dim_b;
```

Comment:
```suggestion
    result &= (gid.get(0) == dim_a) && (gid.get(0) == dim_b);
```

Just to make it a bit easier to read